### PR TITLE
Separated supplemental resources

### PR DESCRIPTION
**Pull request by raibove**
Fixed issue #834

These topics look fine:
https://github.com/Techtonica/curriculum/blob/master/interviewing/week-3-whiteboard-challenges.md
https://github.com/Techtonica/curriculum/blob/master/personal-brand/personal-branding.md

### CODE REVIEW

File: interviewing/whiteboarding.md [hunk 1]
```diff
@@ -113,5 +109,4 @@ Write an input example, and walk it through the function's process and discuss i
 	- [Project Euler](https://projecteuler.net/)
 	- [Code Wars](https://www.codewars.com/)
 - Other resources for practicing
-	- [JS Fiddle](https://jsfiddle.net/) - helpful for practicing frontend coding with Javascript, HTML and CSS
-- Common whiteboarding questions
+	- [JS Fiddle](https://jsfiddle.net/) - helpful for practicing frontend coding with Javascript, HTML and CSS
```
Comment (Kundan28) [inline interviewing/whiteboarding.md]: Javascript -> JavaScript

File: javascript/javascript-4.md [hunk 2]
```diff
@@ -27,25 +27,11 @@ JavaScript is used on the frontend of almost every website. It's also a widely-u
...
-**While Loops**
-
-- [JavaScript While Loops (YouTube video)](https://www.youtube.com/watch?v=PpbFyLTtpWI)
-- [While Loop docs (MDN)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/while)
-- [While Loops (w3schools)](https://www.w3schools.com/js/js_loop_while.asp)
-
 ### Lesson
 
-[JavaScript 4 (20 min video)](https://drive.google.com/file/d/1YL8rU9wY3fh0g4mZM223b7g-PDgk_X5P/view?usp=sharing)
+Video walkthrough of lesson slides [JavaScript 4](https://drive.google.com/file/d/1YL8rU9wY3fh0g4mZM223b7g-PDgk_X5P/view?usp=sharing)
```
Comment (Kundan28) [inline javascript/javascript-4.md]: You can also add video time at all the places.

File: seo/seo.md [hunk 3]
```diff
@@ -72,3 +64,10 @@ Review your own portfolio site, connect it to some of the tools learned in this
 - What is Google Search Console?
 - What are the benefits of generating a SEO-friendly content?
 - What tools do you use for SEO?
+
+### Supplemental Materials
+
+- [Google Search Console](https://gsuite.google.com/products/admin/)
+- [Google Analytics](www.google.com/analytics)
+- [Google My Business](www.google.com/business)
+- [Must-Have Social Meta Tags for Twitter, Google+, Facebook and More](https://moz.com/blog/meta-data-templates-123)
```
Comment (Kundan28) [inline seo/seo.md]: Google + is not active now so you can remove that.

File: writing-readable-code/writing-readable-code.md [hunk 4]
```diff
@@ -53,6 +47,10 @@ Techtonica staff will assign #1 to half of the group and #2 to the other half of
 
 Pair up with a colleague who did not do the same coding challenge as you. Show your pair partner your code from the Independent Practice, but do not explain it to them. Your partner should read the code to themselves, then explain back to you their understanding of what each line of your code is doing. Listen to their feedback and suggestions — they may end up helping you to identify parts of your code that could be improved.
 
+### Supplemental Materials
+
+- [Writing Readable JavaScript (video)](https://www.youtube.com/watch?v=8WF4AjM-XW8)
```
Comment (Kundan28) [inline writing-readable-code/writing-readable-code.md]: You can write the description of the video after the link.

**Comment by alodahl** [comment]
Don't title your commits "Update...". Describe what you actually did.

**Comment by raibove** [comment]
> Don't title your commits "Update...". Describe what you actually did.

Ok. I thought that in almost every commit I updated the time so it would be confusing. Therefore I wrote it in description.

File: accessibility/accessibility.md [hunk 5]
```diff
@@ -56,19 +60,22 @@ Myth: We only need to consider color contrast when designing for vision impairme
...
 
-- [Intro to web accessibility](https://www.creativebloq.com/netmag/simple-introduction-web-accessibility-7116888)
+- [Introduction to Web Accessibility](https://www.creativebloq.com/netmag/simple-introduction-web-accessibility-7116888)
 - [Web experiences of people with disabilities](https://webaim.org/intro/#people)
 - [Google accessibility course (text)](https://developers.google.com/web/fundamentals/accessibility/) 
 - [Google accessibility course (video)](https://www.udacity.com/course/web-accessibility--ud891)
 - [A11Y Style Guide](https://a11y-style-guide.com/style-guide/)
 
+### Check for Understanding
+
```
Comment (alodahl) [inline accessibility/accessibility.md]: Are you solving a check for understanding issue or a materials issue?

**Comment by raibove** [comment]
I tried to make all changes in single commit but was not able to do so. I will do it from next time.

File: accessibility/accessibility.md (hunk 5 above)
Comment (raibove) [reply accessibility/accessibility.md]: No. But for this file the questions where already there in Guided Practice so I shifted them here and also added one question

Comment (alodahl) [reply accessibility/accessibility.md]: ah, thank you.

**Pull request merged**